app/models/bishop.rb

```diff
@@ -1,4 +1,26 @@
 class Bishop < Piece
   scope :piece, -> {where(type: 'Bishop')}
 
+  def valid_move?(position_x, position_y)
+    piece.position_x = x_current
+    piece.position_y = y_current
+    position_x = x_destination
+    position_y = y_destination
+
+    if outside_board?
```

Comment:
you need to pass the coordinates to the method call.. `outside_board?(position_x, position_y)`